src/GenerateDemoData.php

```diff
@@ -105,7 +105,7 @@ private function getDateRange(): array
             Carbon::parse($endDate)
         );
         foreach ($period as $date) {
-            $dateRange[] = $date;
+            $dateRange[] = new Carbon($date);
```

Comment:
OK, perbaikan di titik ini sudah memperbaiki errornya, tapi mungkin kita pakai Carbon::parse saja agar sama seperti kode-kode lain di atasnya.

```suggestion
            $dateRange[] = Carbon::parse($date);
```